File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
        n = get_int("Height: \n");
    while (n < 1 || n > 8);
    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= n - i; j++)
        {
            printf(" ");
        }
        for (int a = 1; a <= i; a++)
        {
            printf("#");
        }
        printf("  ");
        for (int b = 1; b <= i; b++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int
h = get_int("Height:")
while h <= 0 or h > 8:
    h = get_int("Height:")
for i in range(1, h+1):
    for j in range(h-i):
        print(" ", end = "")
    for p in range(i):
        print("#")
print()
